src/renderer/services/file-management-system/index.ts

```diff
@@ -145,6 +147,16 @@ export default class FileManagementSystem {
         lastModifiedInMS: fileLastModifiedInMs,
       },
     });
+    switch (registration.status) {
+      case UploadStatus.RETRY:
+        throw new Error(
+          'Your file uploaded and is safe, but a post-upload task failed on the backend.  Please wait a few moments, and the retry (select the box to the right of your upload, and use the "retry" button.'
```

Comment:
Is it uploaded after just registering with FSS?

Reply:
For a localNas upload, yes.  
But good catch, i need to think about this a bit.

Reply:
This came from the `waitForServerCopy` method which previously was called just after registering.  It checked for these statuses since the upload could transition into them while FUA polled for the status.  I now realize is doesnt make sense to check for it right after registering.  The problem now is that if the upload does transition into these states, fua will simply stop receiving progress updates.  It won't know that it happened, since it is not polling.  I guess the solution is to add a serviceField for fssStatus, so that FUA recieves a SSE when this happens.

Reply:
Oh right! it does update JSS, in the stage field.  So I just need to dispatch an event to fail the FUA upload in those cases so that the user can retry later.

Reply:
Looks like this is possibly handled already, but i need some in person review.